Gemfile

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.1.4'
```

Comment:
Also, all gems have a comment before the actual dependency. Obviously not in this PR because you didn't add any, but try to add one yourself for any new gem you add.